web/src/screens/library/LibraryPageScreen/blocks/LibraryPageTableBlock/column.ts

```
@@ -0,0 +1,201 @@
+      ? getDefaultBlockConfig(schema)
+      : processBlockConfig(schema, block.config);
+
+  const columns: ColumnValue[] = config.columns.reduce((prev, column) => {
+    if (column.hidden) {
+      return prev;
+    }
+
+    if (column._fixedFieldName) {
+      if (column._fixedFieldName === "link") {
+        return [
+          ...prev,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)